apps/address-service/domains/common/utils/services/search/plugins/SearchByProvider.js

```diff
@@ -0,0 +1,75 @@
+const { AbstractSearchPlugin } = require('@address-service/domains/common/utils/services/search/AbstractSearchPlugin')
+const { Address } = require('@address-service/domains/address/utils/serverSchema')
+const { generateAddressKey } = require('@address-service/domains/common/utils/addressKeyUtils')
+const { SearchProviderDetector } = require('@address-service/domains/common/utils/services/search/SearchProviderDetector')
+const get = require('lodash/get')
+
+class SearchByProvider extends AbstractSearchPlugin {
+
+    /**
+     * @param s
```

Comment:
Please provide type of this param if possible

Reply:
Fixed.
Anyway, jsdoc is inherited from the parent class.